CMakeLists.txt

```diff
@@ -64,7 +65,7 @@ add_subdirectory(tests/cpp)
 
 install(FILES LICENSE DESTINATION docs/licensing COMPONENT licensing_genai RENAME LICENSE-GENAI)
 install(FILES third-party-programs.txt DESTINATION docs/licensing COMPONENT licensing_genai RENAME third-party-programs-genai.txt)
-set(CPACK_ARCHIVE_COMPONENT_INSTALL ON)
+set(CPACK_ARCHIVE_COMPONENT_INSTALL ${GENAI_CPACK_ARCHIVE_COMPONENT_INSTALL})
```

Comment:
maybe we can just use:
```
if(NOT DEFINED CPACK_ARCHIVE_COMPONENT_INSTALL )
    set(CPACK_ARCHIVE_COMPONENT_INSTALL  ON)
endif()
```
instead of extra option with different name

Reply:
I tried that as well. It doesn't work

Reply:
See https://github.com/openvinotoolkit/openvino/pull/25867

With that PR we can override this option via cmake cmd and this PR is not required